src/libraries/apiclient/apiclientcore.js

```diff
@@ -150,27 +150,51 @@ define(["events", "appStorage"], function(events, appStorage) {
             if (messageIdsReceived[messageId]) return;
             messageIdsReceived[messageId] = !0
         }
-        if ("UserDeleted" === msg.MessageType) instance._currentUser = null;
-        else if ("UserUpdated" === msg.MessageType || "UserConfigurationUpdated" === msg.MessageType) {
+        if ("UserDeleted" === msg.MessageType) {
```

Comment:
All the API client stuff should be mirrored to https://github.com/jellyfin/jellyfin-apiclient-javascript, as #1007 removes all the API client files from this repository.

Reply:
Done, [related PR](https://github.com/jellyfin/jellyfin-apiclient-javascript/pull/44).